docs/products/flink/getting-started.rst

```diff
@@ -1,12 +1,74 @@
 Getting started
 ===============
 
-Aiven for Flink is available in the `Aiven console <https://console.aiven.io>`_.
+The first steps in using Aiven for Apache Flink are to create a service and a Flink job. You can do this in the Aiven web console or with the Aiven CLI.
```

Comment:
Can we add the links to Aiven web console and Aiven CLI?

Reply:
Good point - done ;)